tabs/osd.html

```diff
@@ -46,6 +50,12 @@ <h1 class="tab_title" data-i18n="tabOSD"></h1>
                     <div class="spacer_box">
                         <div class="video-types"></div>
                     </div>
+                    <div class="spacer_box settings">
```

Comment:
This is a nice touch. Though it would be much better is the state was stored in Configurator. You can use the chrome storage for this.

Reply:
Tell me more about this and how to make it work (html, css, js, etc. are not my 1st languages)